.github/workflows/early-access.yml

```diff
@@ -23,20 +23,13 @@ on:
 jobs:
   # Build native executable per runner
   build:
-    name: 'Build with Graal on ${{ matrix.os }}'
+    name: 'Build with Graal ${{ matrix.version }} on ${{ matrix.os }}'
     if: github.repository == 'kcctl/kcctl' && startsWith(github.event.head_commit.message, '🏁 Releasing version') != true && startsWith(github.event.head_commit.message, '⬆️  Next version') != true
     strategy:
       fail-fast: true
       matrix:
+        version: ['21.3.0', '21.3.0', '21.3.0']
```

Comment:
Why testing the same version three times? I think this would test 21.3.0 three times for each element of the "os" axis.

Reply:
yes, apologize let me fix it